app/viewmodels/tasklist/VatRegistrationTaskList.scala

```diff
@@ -105,22 +123,33 @@ class VatRegistrationTaskList @Inject()(aboutTheBusinessTaskList: AboutTheBusine
     TaskListSection(
       heading = messages("tasklist.vatRegistration.heading"),
       rows = Seq(
-        Some(buildGoodsAndServicesRow.build(vatScheme)),
+        Some(goodsAndServicesRow.build(vatScheme)),
         resolveBankDetailsRow(vatScheme).map(_.build(vatScheme)),
         resolveVATRegistrationDateRow(vatScheme).map(_.build(vatScheme)),
-        Some(buildVatReturnsRow.build(vatScheme))
-      ).flatten
+        Some(vatReturnsRow.build(vatScheme)),
+        resolveFlatRateSchemeRow(vatScheme)
+    ).flatten
     )
   }
 
   private def resolveBankDetailsRow(vatScheme: VatScheme)(implicit profile: CurrentProfile) = {
     if (Seq(NETP, NonUkNonEstablished).exists(vatScheme.partyType.contains)) None else Some(bankAccountDetailsRow)
   }
 
+  private def resolveFlatRateSchemeRow(vatScheme: VatScheme)(implicit profile: CurrentProfile) = {
+    if (vatScheme.vatApplication.flatMap(_.turnoverEstimate).exists(_ > 150000L) ||
+      vatScheme.eligibilitySubmissionData.map(_.registrationReason).exists(GroupRegistration.equals)
+    ) {
+      None
```

Comment:
Can we add a test case for this flow pls, so that we verify no FRS row being shown for the conditions mentioned here. Thanks!